#!/usr/bin/env python3

def throw_custom_exception():
    raise Exception('This is an exception')

if __name__ == '__main__':
    throw_custom_exception()
